NetSDK/Services/Impressions/Classes/BlockingQueue.cs

```diff
@@ -21,6 +21,10 @@ public BlockingQueue(int maxSize)
 
         public bool HasReachedMaxSize()
         {
+            if (maxSize == -1)
```

Comment:
I think that if I leave deafult value (-1) I want my caché with unlimited space, so there is no way to know if it has reached is max size. Maybe I didn't get the meaning of "-1" right